src/coreclr/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.Shared.xml

```diff
@@ -36,6 +36,10 @@
     <type fullname="Internal.Runtime.InteropServices.InMemoryAssemblyLoader">
       <method name="LoadInMemoryAssembly" />
     </type>
+
+    <type fullname="System.Diagnostics.StackFrame">
```

Comment:
Why do we need to root this? It doesn't look like the runtime relies on the presence of this method.